generator.yaml

```diff
@@ -120,6 +124,10 @@ resources:
           resource: SecurityGroup
           service_name: ec2
           path: Status.ID
+      SnapshotIdentifier:
+        from:
+          operation: RestoreDBClusterFromSnapshot
+          path: SnapshotIdentifier
```

Comment:
What happens if the user changes the value of `Spec.SnapshotIdentifier`? :)

Reply:
Yeah, like you said, it will end up with nothing happening if customer update it. 
I will add a tutorial part about how customer do restore db instance/cluster from snapshot and mention customer need be careful not to update this `SnapshotIdentifier` after it's set